data/cube/aggregate/AverageStrictAggregator.js

```diff
@@ -10,15 +10,21 @@ import {Aggregator} from '@xh/hoist/data/cube/aggregate/Aggregator';
 export class AverageStrictAggregator extends Aggregator {
 
     aggregate(rows, fieldName) {
-        let total = null;
+        let strict = true,
+            total = null,
+            count = 0;
```

Comment:
Had to add this back, we can't divided by number of rows (which might be 1), we need to divided by number of children